### ISSUE TITLE
Website fails to generate JSONSchema: "Value cannot be null. (Parameter 'references')"

### ISSUE THREAD

**Issue by apsz3** [json-everything/json-everything#832]
### Nuget Package

json-everything.net (playground)

### Package Version

web

### Operating System

None

### .Net Target (if relevant, please specify the version in the description)

None

### A clear and concise description of the bug.

This code previously ran fine a few weeks ago; today, when I try to generate a JSONSchema from this, I get the error "Value cannot be null. (Parameter 'references')" in the output box where the schema generation usually rendered. 

### What did you expect?

Working generation

### Please add test code or steps to reproduce the behavior.

```
public class EEE
{

    public Guid A { get; set; }
    

    public DateTime? B { get; set; }
    

    public DateTime? C { get; set; }
    

    public int? D { get; set; }
    

    public int? E { get; set; }
    

    public int? F { get; set; }
    

    public int? G { get; set; }
    

    public int? H { get; set; }


    public byte I { get; set; }

    
    public long J { get; set; }
    
    
    public int K { get; set; }

    public int L { get; set; }
    
    public DateTime M { get; set; }

}
```

### Code of Conduct & Contributor Guidelines

- [X] I agree to follow this project's Code of Conduct and Contribution Guidelines.

**Comment by gregsdennis** [comment]
Yeah, the website seems to have some problems since I updated to net9. I haven't had the time to look into it. Thanks for reporting this.

**Comment by gregsdennis** [comment]
The problem seems to be down to the site being unable to find the DLLs.  This apparently changes with every release, I've got to figure this out again.

Question asked: https://stackoverflow.com/q/79325262/878701

**Comment by gregsdennis** [comment]
StackOverflow didn't work out.  Got the answer in the .Net repo.

https://github.com/dotnet/aspnetcore/issues/59750

Everything seems to be working now.
